ruby_review_week_four.rb

```diff
@@ -1,37 +1,135 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+can store data.
+
 
 # 2. What Ruby method can you use to make a loop from an array?
+ruby each loop
+
 
 # 3. In Ruby, what are booleans?
 
+a boolean refers to a value of either true or false, 
 # 4. How do classes relate to objects?
 
 # 5. What is an instance variable?
+class Person
+  def initialize(name)
+    @name = name
+  end
+end
+@name is a new type of variable, called an “instance variable”.
```

Comment:
remember that an instance variable is an attribute of an object, attribute is the key word i'm looking for, you could also talk about instance variables having scope within all instance methods